ouroboros-network-api/src/Ouroboros/Network/PeerSelection/LedgerPeers/Util.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE BangPatterns     #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Ouroboros.Network.PeerSelection.LedgerPeers.Util
+  ( bigLedgerPeerQuota
+  , accBigPoolStake
+  , reRelativeStake
```

Comment:
It's probably worth to also re-export `AccPoolStake`, `PoolStake` and `RelayAccessPoint`, e.g.

```suggestion
  ( bigLedgerPeerQuota
  , accBigPoolStake
  , reRelativeStake
    -- Re-exports
  , AccPoolStake (..)
  , PoolStake (..)
  , RelayAccessPoint (..)
```
It might simplify imports.